postgres-appliance/create_user_functions.sql

```diff
@@ -138,6 +138,21 @@ COMMENT ON FUNCTION drop_user(text) IS 'Drop a human or application user.  Inten
 Roles (= users) that own database objects cannot be dropped.';
 
 
+CREATE OR REPLACE FUNCTION drop_role(username text)
+ RETURNS void
+ LANGUAGE sql
+AS $function$
+SELECT user_management.drop_user(username);
+$function$
+SECURITY DEFINER;
+
+REVOKE ALL ON FUNCTION drop_role(text) FROM public;
+GRANT EXECUTE ON FUNCTION drop_role(text) TO admin;
+
+COMMENT ON FUNCTION drop_role(text) IS 'Drop a human or application user.  Intended for cleanup (either after team changes or mistakes in role setup).
```

Comment:
Maybe it makes sense to mention that `drop_role` is merely an alias for `drop_user`, just to make it clear?